Src/Particle/AMReX_Particle.H

```diff
@@ -93,27 +94,75 @@ namespace amrex {
         idcpu &= ~(uint64_t(1) << 63);
     }
 
+    template<typename T_SIMD, typename T_Mask>
+    AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
+    void make_invalid (T_SIMD & idcpu, T_Mask const & mask) noexcept {
+        // RHS mask: 0111...
+        if constexpr (std::is_same_v<T_Mask, bool>) {
+            // scalar
+            static_assert(std::is_same_v<T_SIMD, uint64_t>, "make_invalid: forgot template on wrapper?");
+            if (mask) { make_invalid(idcpu); }
+        } else {
+#ifdef AMREX_USE_SIMD
+            // SIMD
+            static_assert(!std::is_same_v<T_SIMD, uint64_t>, "make_invalid: forgot template on wrapper?");
+            // .__cvt() because of: https://github.com/VcDevel/std-simd/issues/41
+            simd::stdx::where(mask.__cvt(), idcpu) &= ~(T_SIMD{1} << 63);
```

Comment:
Memo to myself: in ImpactX tests, this part seems to cause issues. Not clear if an issue in AMReX or in the ImpactX usage if vectorized idcpu.

Code here looks correct...